src/osemgrep/networking/Unit_Networking.ml

```diff
@@ -17,60 +17,33 @@
 (* Prelude *)
 (*****************************************************************************)
 
-module Http_helpers = Http_helpers.Make (Lwt_platform)
-
 let t = Testo.create
 
 (*****************************************************************************)
 (* Helpers *)
 (*****************************************************************************)
 
-let get_and_check caps url =
-  Logs.debug (fun m -> m "GET %s" url);
-  let uri = Uri.of_string url in
-  let response = Http_helpers.get caps#network uri in
-  (* Check OK Status *)
-  match response with
-  | Ok (body, _) -> body
-  | Error (e, _) -> failwith (Printf.sprintf "Error (%s): %s" url e)
-
-let post_and_check caps url body =
-  Logs.debug (fun m -> m "POST %s" url);
-  let uri = Uri.of_string url in
-  let response = Http_helpers.post ~body caps#network uri in
-  (* Check OK Status *)
-  match response with
-  | Ok body -> body
-  | Error (_, msg) -> failwith (Printf.sprintf "Error (%s): %s" url msg)
-
 let get_and_check_lwt caps url =
```

Comment:
If we're dropping `_async` from `get` & `post` seems like we should do it here too if there are no longer blocking versions.

Reply:
fair